### PR TITLE
FT2-124 Dashboard

### PR DESCRIPTION
**Pull request by lazcool**
Adds initial content approval dashboard part and widget.

Also updates our orchard core packages with a fix applied to OrchardCore.AdminDashboard. (Our [fork has been updated](https://github.com/SkillsFundingAgency/OrchardCore/tree/monaco_editor), and a [PR](https://github.com/OrchardCMS/OrchardCore/pull/8620) raised against upstream.)

### CODE REVIEW

File: DeveloperNotes.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 #ToDo
 
+* make open site open site (for env)
```
Comment (andycyberian) [inline DeveloperNotes.md:3]: Not 100% sure what this means.

**Review by andycyberian — COMMENTED** [review]
All looks good, just the one comment which may just be me not understanding.

File: DeveloperNotes.md (hunk 1 above)
Comment (lazcool) [reply DeveloperNotes.md:3]: There's a visit site button..
image

We should make it open the current environment's National Careers site.

**Review by andycyberian — APPROVED** [review]
All looks good. I see that approval statuses has been expanded on which may affect some work on content approval but we can discuss this in the refinement session.

**Pull request merged**